Appraisals

```diff
@@ -1,17 +1,26 @@
-%w(4.0 4.1 4.2).each do |version|
-  appraise version do
-    gem "actionpack", "~> #{version}.0"
-    gem "activerecord", "~> #{version}.0"
-    gem "railties", "~> #{version}.0"
+[ '4.0', '4.1', '4.2' ].each do |ver|
+  appraise "rails-#{ver}" do
+    gem 'actionpack',   "~> #{ver}.0"
+    gem 'activerecord', "~> #{ver}.0"
+    gem 'railties',     "~> #{ver}.0"
+    gem 'rack',         '~> 1.5'
   end
 end
 
-appraise "edge" do
-  git "https://github.com/rails/rails.git" do
-    gem "actionpack"
-    gem "activerecord"
-    gem "railties"
+appraise 'rails-5.0' do
+  gem 'actionpack',   '>= 5.0.0.alpha', '< 5.1'
+  gem 'activerecord', '>= 5.0.0.alpha', '< 5.1'
+  gem 'railties',     '>= 5.0.0.alpha', '< 5.1'
+  gem 'rack',         '>= 2.0.0.alpha', '< 3'
```

Comment:
I mistyped; Yeah, I wonder how/why it's picking up that dependency given it's a prerelease version. Let me experimenting with it a bit.